advocacy_docs/partner_docs/CommVaultGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Commvault'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Commvault       |
+| **Partner Product**   |  Commvault Express       |
+| **Web Site**      |    https://www.commvault.com/   |
+| **Version & Platform**   |  Commvault Express 11.24: Microsoft Windows Server 2019 Standard      |
+| **Product Description**      |  Commvault provides data protection and data management features.     |
```

Comment:
Better Product Description:
Wherever your data resides, ensure availability via a single interface with Commvault® Backup & Recovery. Say goodbye to costly data loss scenarios, segregated data silos, missing recovery SLAs and inefficient scaling. Empower your team to do more.